src/WebSdk/Publish/Package/Microsoft.NET.Sdk.Publish.csproj

```diff
@@ -53,5 +43,12 @@
     </Content>
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="$(RepoRoot)\src\WebSdk\Publish\Tasks\Microsoft.NET.Sdk.Publish.Tasks.csproj" ExcludeAssets="All" />
```

Comment:
@joeloff 
Had to add P2P reference to get the build ordering correct. This is actually not needed for building. Forcing a solution build order doesn't seem to fix it. 

Will look at a better way to handle it in a subsequent PR.